pyproject.toml

```diff
@@ -42,7 +42,8 @@ dependencies = [
     "statsmodels>=0.13.0",
     "numpy>=1.23",
     "matplotlib>=3.2.0,<4.0",
-    "tqdm>=4.60"
+    "tqdm>=4.60",
+    "regex>=2024.5.15"
```

Comment:
Why do we need `regex`? Is Python's builtin `re` insatisfactory? An honest question..